docs/reference/commandline/index.md

```diff
@@ -29,7 +29,7 @@ read the [`dockerd`](dockerd.md) reference page.
 |:--------|:-------------------------------------------------------------------|
 | [dockerd](dockerd.md) | Launch the Docker daemon                             |
 | [info](info.md) | Display system-wide information                            |
-| [inspect](inspect.md)| Return low-level information on a container or image  |
+| [inspect](inspect.md)| Return low-level information by one or multiple name or ID |
```

Comment:
We discussed this at some point, and we think that for the _index_, it's not needed to explain that one _or multiple_ objects can be passed to some commands, so those changes are not needed (on this line, and the other ones)